roles/homer/templates/config.yml.j2

```diff
@@ -52,97 +52,141 @@ links:
 services:
   - name: "Deployment"
     icon: "fas fa-wrench"
+    {% if homer_url_ara or homer_url_flower or homer_url_netbox or homer_url_vault %}
```

Comment:
Maybe wrap the whole "- name: Deployment" block in this if? That would get rid of the "else" block.

Also, shouldn't these checks also be "| length > 0" each?